models/demos/wormhole/llama31_8b/tt/llama_attention.py

```diff
@@ -429,7 +427,7 @@ def forward_prefill(self, xs_11SH, attn_masks, rot_mats, transformation_mats, us
         q_heads_84SD = ttnn.reshape(
             q_heads_1QSD, [self.n_local_kv_heads, self.n_local_heads // self.n_local_kv_heads, -1, self.head_dim]
         )
-        attn_output_84SD = ttnn.experimental.operations.primary.transformers.scaled_dot_product_attention(
+        attn_output_84SD = ttnn.scaled_dot_product_attention(
```

Comment:
should be ttnn.transformer.scaled_dot_product_attention ?

Reply:
You're right, this is fixed in my local branch, gonna push it in a moment.